import-service/src/handlers/importProductsFile.ts

```diff
@@ -0,0 +1,30 @@
+import { APIGatewayProxyHandler } from "aws-lambda";
+import * as AWS from "aws-sdk";
+
+const BUCKET = process.env.BUCKET;
+
+export const importProductsFile: APIGatewayProxyHandler = async (event) => {
+  const fileName = event.queryStringParameters.name;
+  const filePath = `uploaded/${fileName}`;
+  const s3 = new AWS.S3({ region: "eu-west-1" });
+  const params = {
+    Bucket: BUCKET,
+    Key: filePath,
+    Expires: 60,
+    ContentType: "text/csv",
+  };
+
+  return new Promise((res, rej) => {
```

Comment:
simplify and using `async/await` `const url = await s3.getSignedUrlPromise('putObject', params);`